tinygrad/renderer/cstyle.py

```diff
@@ -189,7 +189,7 @@ class ClangRenderer(CStyleLanguage):
 
   # language options
   buffer_suffix = " restrict"
-  type_map = {dtypes.bool:"_Bool", dtypes.half:"__fp16"}
+  type_map = {dtypes.bool:"_Bool", dtypes.half:"__fp16",dtypes.int8: "int8_t"}
```

Comment:
I know it's minor, but please pay attention to spacing.

Reply:
fixed